src/vim9class.c

```diff
@@ -3841,6 +3841,34 @@ object_len(object_T *obj)
     return tv_to_number(&rettv);
 }
 
+/*
+ * Return TRUE when two objects have exactly the same values.
+ */
+    int
+object_equal(
```

Comment:
This function is very similar to the typval_compare_object() function.  Can you modify typval_compare_object()
to use object_equal()?

Reply:
That's already done, isn't it?

Reply:
Yes, typval_compare_object() uses now `object_equal()`